crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -1329,19 +1390,12 @@ impl Loader<ParentVersionKey> for Db {
             .map(|(group_key, ids)| {
                 self.execute(move |conn| {
                     async move {
-                        let stored: Vec<StoredHistoryObject> = conn
+                        let stored: Vec<StoredObjectVersion> = conn
                             .results(move || {
-                                use objects_history::dsl as h;
                                 use objects_version::dsl as v;
 
-                                h::objects_history
-                                    .inner_join(
-                                        v::objects_version.on(v::cp_sequence_number
-                                            .eq(h::checkpoint_sequence_number)
-                                            .and(v::object_id.eq(h::object_id))
-                                            .and(v::object_version.eq(h::object_version))),
-                                    )
-                                    .select(StoredHistoryObject::as_select())
+                                v::objects_version
+                                    .select(StoredObjectVersion::as_select())
```

Comment:
nit: I'm not sure you need to make this `select` explicitly if `StoredObjectVersion` is all the fields in `objects_version`.